linera-chain/src/chain.rs

```diff
@@ -693,6 +693,46 @@ where
 
         let chain_id = self.chain_id();
         assert_eq!(block.chain_id, chain_id);
+        // The first incoming message of any child chain must be `OpenChain`. A root chain must
+        // already be initialized
+        if block.height == BlockHeight::ZERO
+            && self
+                .execution_state
+                .system
+                .description
+                .get()
+                .map_or(true, |description| description.is_child())
+        {
+            let Some(in_bundle) = block
+                .incoming_bundles
+                .first()
+                .filter(|in_bundle| in_bundle.action == MessageAction::Accept)
+            else {
+                return Err(ChainError::InactiveChain(chain_id));
+            };
+            let Some(posted_message) = in_bundle.bundle.messages.first().filter(|pm| {
+                println!("{:#?}", pm.message);
```

Comment:
I guess this `println!` should be removed? (Or replaced with `trace!`?)

Reply:
Sorry, too much debugging today! :facepalm: